### PR TITLE
#451 Update Windows & Mac

### PR DESCRIPTION
**Pull request by dthiesse**
Got the Windows doc done and fixed a few things I missed on Mac (most importantly the mysql client).

### CODE REVIEW

File: setup/set_up_windows.md [hunk 1]
```diff
@@ -4,168 +4,164 @@ This document provides the steps to install the software required to begin the e
...
+8.	Install IBM Cloud (BLuemix) CLI
+9.	Install the IBM Container Service plugins
+10. Install the Kubernetes kubectl CLI
+11. Install the MYSQL client
+12. Install Cygwin
+
 
 ## Part 1: Install Java JDK 1.8
 
 1.	In a browser, open https://www.oracle.com
```
Comment (palistra) [inline setup/set_up_windows.md:21]: Any chance we can use the IBM JDK?  Not sure if Oracle's will cause a problem for students who install Eclipse and WAS for other classes.

**Review by palistra — COMMENTED** [review]
You go into some detail for some items (like nodejs) but not for others (Docker, Cygwin).  I thought they would all be like what is in here for Docker and Cygwin, just a pointer to the software and tell students to follow the install instructions.  This would avoid having to update this document if the installation details change?
Let me know what you think, thanks!

**Comment by dthiesse** [comment]
Good point on Docker, Cygwin, and node.js. I'll look at those parts again and try not to have too much detail directly in the document. As for the question about Java, I think we have been using Oracle's version for some time now, at least in our setup docs. Is there a difference between Oracle's JDK and IBM's? I honestly do not know. I think we (at least I) have been using Oracle's JDK for some time with Eclipse and WAS/Liberty with no ill effects.

**Comment by palistra** [comment]
Re: Java - the last I heard, we were not supposed to download or use Oracle JDK on IBM machines unless the Oracle JDK provided something the IBM JDK did not.  I have to admit that was a while ago and things may have changed.  I will ask on the Standup if anyone else heard.

**Comment by dthiesse** [comment]
@palistra , since the Java question has been resolved, I'm going to go back to work on the Mac and Windows docs next, to remove unnecessary detail (leave it up to instructions on the web site) and to make them consistent with each other in terms of content and detail level. If you want to merge this request, I can just do another one when I commit and push, so our audience sees the progress we have made so far.

**Pull request merged**